apps/challenges/models.py

```diff
@@ -167,6 +167,8 @@ def __init__(self, *args, **kwargs):
     desired_worker_instance = models.IntegerField(
         null=True, blank=True, default=1
     )
+    # Enable/Disable EFS creation for the challenge
+    enable_efs = models.BooleanField(default=False)
```

Comment:
We should also move this to the challenge phase, so suppose if there is a case where one phase requires the `EFS` and other doesn't we would end up creating `EFS` for both the phases. What do you think?

Reply:
We can move it to the challenge phase but if EFS is not being used there won't be any cost associated with it https://aws.amazon.com/efs/pricing/.

Reply:
If we are moving enable flag to the phase model we should move all other meta data like security_group, mount_targets, etc. But I wanted to avoid storing cluster related details in phase model